src/daft-local-execution/src/sinks/aggregate.rs

```diff
@@ -38,44 +30,22 @@ impl AggregateSink {
 impl BlockingSink for AggregateSink {
     #[instrument(skip_all, name = "AggregateSink::sink")]
     fn sink(&mut self, input: &Arc<MicroPartition>) -> DaftResult<BlockingSinkStatus> {
-        if let AggregateState::Accumulating(parts) = &mut self.state {
```

Comment:
We should still keep the phases in for AggregateState. This is meant to be a placeholder for when we have a folding aggregation.